includes/admin/downloads/metabox.php

```diff
@@ -591,8 +612,9 @@ function edd_render_products_field( $post_id ) {
 										<span class="edd-product-file-reorder edd-draghandle-anchor dashicons dashicons-move"  title="<?php printf( __( 'Click and drag to re-order bundled %s', 'easy-digital-downloads' ), edd_get_label_plural() ); ?>"></span>
 										<input type="hidden" name="edd_bundled_products[<?php echo $index; ?>][index]" class="edd_repeatable_index" value="<?php echo $index; ?>"/>
 									</div>
-									<div class="edd-bundled-product-item">
-										<span class="edd-repeatable-row-setting-label"><?php printf( __( 'Select %s:', 'easy-digital-downloads' ), edd_get_label_singular() ); ?></span>
+									<div class="edd-form-group edd-bundled-product-item">
+										<label for="edd_bundled_products_<?php echo esc_attr( $index ); ?>" class="edd-form-group__label edd-repeatable-row-setting-label"><?php printf( esc_html__( 'Select %s:', 'easy-digital-downloads' ), edd_get_label_singular() ); ?></label>
+										<div class="edd-form-group__control">
 										<?php
 										echo EDD()->html->product_dropdown( array(
```

Comment:
Do we want an `edd-form-group__input` class here, too?

Reply:
I can add it to the array, but it looks like the product dropdown overwrites any class I'll pass to it. Adding it anyway in case that's deemed a bug.